src/app/api.js

```diff
@@ -0,0 +1,33 @@
+const userInput = document.querySelector('#name');
```

Comment:
- [ ] Good job 👍. Your app is working as expected and there are no errors in your code. However, You forgot to add all information in your `README.md` file. Please kindly update the file 😊🍀

Your readme
![image](https://user-images.githubusercontent.com/69011963/147351246-d922b123-6cb9-4f58-9fc9-f07910d375d7.png)